package.json

```diff
@@ -116,6 +123,13 @@
           "settings.enabled"
         ]
       },
```

Comment:
Shouldn't this permission be hidden too since now we have `ui-eholdings.settings.enabled` which is meant to be shown in the permissions list?